source/README.adoc

```diff
@@ -0,0 +1,142 @@
+
+:toc:
+:toc-placement!:
+= RabbitMQ Source
+
+The RabbitMQ source translates messages on a RabbitMQ exchange to CloudEvents
+over HTTP for use with Knative Eventing. The Source can bind to an existing
+exchange or create a new one, as needed.
+
+toc::[]
+
+
+== Published Events
+
+All messages received by the Source will be published with the following schema:
+
+.Table Event Attributes
+|===
+| Attribute | Value | Notes
+
+| `type` | `dev.knative.rabbitmq.event` |
+| `source` | `/apis/v1/namespace/*$NS*/rabbitmqsources/*$NAME*#*$TOPIC*`
+   | `NS`, `NAME` and `TOPIC` are derived from the Source configuration.
+| `id` | a unique id | This uses the MessageId if available, and a UUID otherwise.
+| `subject` | the message's id | empty string if no message ID is present.
+| `datacontenttype` | `application/json` | Currently static.
+| `key` | the message's id | empty string if no message ID is present.
+|===
+
+The event's payload will be set to the data content of the message.
+
+=== Samples
+
+(in CloudEvents JSON format):
+[source,json]
+----
+{
+  "specversion": "1.0",
+  "type": "dev.knative.rabbitmq.event",
+  "source": "/apis/v1/namespaces/default/rabbitmqsources/rabbitmq-source",
+  "id": "f00c1f52-33a1-4d3d-993f-750f20c804da",
+  "time": "2020-12-18T01:15:20.450860898Z",
+  "subject": "",
+  "datacontenttype": "application/json",
+  "key": "",
+  "data": "Hello rabbitmq!"
+}
+----
+
+== Creating and Managing Sources
+
```

Comment:
Seems like there's an extra line here

Reply:
There's a thin rule after each H2 section -- you can see the same thing at "Administration" and "Published events". Note that both of those have text immediately following, rather than a table.